util-bukkit/src/main/i18n/templates/strings.properties

```diff
@@ -1045,415 +1033,385 @@ observer.tools.flyspeed.hyperspeed = Hyperspeed
 observer.tools.visibility.shown = Shown
 observer.tools.visibility.hidden = Hidden
 
-teleportTool.displayName = Teleport Tool
-editWand.displayName = Edit Wand
-
-player.inventoryPreview.noPotionEffects = No potion effects
-player.inventoryPreview.notViewable = Player's inventory is not currently viewable
-player.inventoryPreview.potionEffects = Potion Effects
-player.inventoryPreview.hungerLevel = Hunger level
-player.inventoryPreview.healthLevel = Health level
-player.inventoryPreview.specialAbilities = Special Abilities
-specialAbility.flying = Flying
-specialAbility.doubleJump = Double Jump
+observer.teleportTool = Teleport Tool
+observer.editWand = Edit Wand
+
+observer.inventoryPreview.noPotionEffects = No potion effects
```

Comment:
How about `preview` instead of `inventoryPreview`

Reply:
Sounds nicer, but do you not think it's too little descriptive?